src/main/java/frc/team166/robot/RobotMap.java

```diff
@@ -39,4 +41,22 @@
         public final static int XboxAbutton = 1;
         public final static int XboxXbutton = 3;
     }
+
+    public static class Encoders {
+        //changes Encoder ports into integers
+        public final static int LIFT_A = 1;
+        public final static int LIFT_B = 2;
+    }
+
+    public static class Solenoids {
+        //changes Solenoid ports into integers
+        public final static int LiftTransmission = 1;
+        public final static int LiftTransmission2 = 2;
+    }
+
+    public static class DigitalImputs {
+        //changes digital imput ports into integers
+        public final static int LIMIT_SWITCH_LIFT_BOTTOM = 1;
+        public final static int LIMIT_SWITCH_LIFT_TOP = 2;
```

Comment:
I would suggest renaming these to, LIFT_LIMIT_SWITCH_BOTTOM, LIFT_LIMIT_SWITCH_TOP